fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/socialmedia/SocialMediaViewModel.java

```
@@ -131,8 +131,10 @@ public void setLaoName(String laoName) {
   public Single<MessageGeneral> sendChirp(
       String text, @Nullable MessageID parentId, long timestamp) {
     Log.d(TAG, "Sending a chirp");
-    Lao lao = getCurrentLao();
-    if (lao == null) {
+    final Lao lao;
+    try {
+      lao = getCurrentLao();
+    } catch (UnknownLaoException e) {
```

Here you're hiding the exception instead of rethrowing it, no?